### ISSUE TITLE
SASS Support

### ISSUE THREAD

**Issue by bfintal** [shff/parcel-plugin-purifycss#2]
It would be great if this was extended to support SASS asset types

**Comment by shff** [comment]
I did some research and tried some stuff but I still haven't figured how to make that work in Parcel.

However, I believe it would be possible to run PurifyCSS over the final, generated CSS, instead of running it once per CSS asset. In fact, that would probably speed things up for bigger projects.

I'll give it a try!

**Comment by sheerun** [comment]
@shff How to do it with sass?

**Comment by shff** [comment]
Hi @bfintal and @sheerun ! I haven't been able to make it work with Parcel, but I made a PostCSS plugin that does the same thing as PurifyCSS. I published it here: https://github.com/shff/postcss-remove-unused-css

The plugin doesn't depend on PurifyCSS (Purify is unmaintaned, and so is its CSS parser/compiler). I also believe that a PostCSS plugin is more appropriate for such things than a Parcel plugin. I think it will work fine with SASS files, since PostCSS runs after SASS compilation. I have tested it with projects that use LESS.
